### ISSUE TITLE
ReC fusion reactor wierd lag, performance lower than expexted and steam line derpinnes

### ISSUE THREAD

**Issue by pajrle** [ReikaKalseki/Reika_Mods_Issues#2929]
I recently built my first fusion reactor and ran into two big issues that I am quite sure are not my error. First of them is, that after starting reactor it makes client side very laggy. It appears like block lag, but my local server is always on 100/99% tps (I check with /cofh tps command). I tried to run my world to run on client only and lag behaved same. Other issue is that steam lines source of heat detection still reports low numbers of fission even though I run boilers 100% of fusion (reactor source types: something like 4.2069420E-5%). This might be linked to other issue: The reactor could not start even 1 hp turbine alone, even though it was 4 injector design, and almost all boilers stay above 100C all the time. I tried nbtedit high amount of steam into steam line and the reactor is than able to run five hp turbines while building up steam in steam lines very quickly. Lag also gets quite lower after I nbtedit more steam into steam lines. That suggests that steam lines might be source of lag for me. I am running latest version of your mods in N&B modpack.

**Comment by MalTeeez** [comment]
I can agree with the weird lag, but for the steam can you send a screenshot on your routing to turbines?

**Comment by pajrle** [comment]
Here is your screenshot:[](url)
![2021-11-07_11 46 03](https://user-images.githubusercontent.com/56485090/140641831-ab5c95db-4827-4541-9169-47b50a44fe4d.png)
Removing pipe pump does not change anything.

**Comment by MalTeeez** [comment]
Did you try disconnecting the lines from turbines, filling up the turbine-side steam lines with at least 8.000m^3 each, letting the turbine structure reform and then reconnecting the lines?

**Comment by pajrle** [comment]
No. And even if that work, I dont think that is not how it should work.
Update: steam boilers/neutron absorbers lag out after some while and stop producing steam at all.

**Comment by ReikaKalseki** [comment]
HP turbines have a very high startup cost - MalTeeez's suggestion makes sense given that you are using five of them.

**Comment by pajrle** [comment]
Ok. But I thought 4 injector fusion reactor should be able to start atleast 1 hp turbine instantly. Also, do you need any more info on the lag issue ?

**Comment by XenonDB** [comment]
> Ok. But I thought 4 injector fusion reactor should be able to start atleast 1 hp turbine instantly. Also, do you need any more info on the lag issue ?

A HP turbine consume 35 ~ 40m/t steam to run. If you supply steam lower than the amount, it just consume it and do nothing.
The long steam line seems a reason because longer line will take more time to balance steam storage to 40m. (Turbine only takes steam from line that directly connects to it)
If you are sure your steam production is over 5 * 35 ~ 40m/t (= 175~200 m/t), try reduce the length of your line (such as using world rift from ChromatiCraft, which is a tesseract like block and can transport lots type of things including steam) or let steam accumulated in line first then connect to turbine.

**Comment by pajrle** [comment]
I came here to report lag issue and heat source detection, not ask for advice. Also I tried to accumulate steam and it did not help.

**Comment by QuantumZizo** [comment]
Im having the same problem currently, and it only started happening AFTER i added four injectors

**Comment by QuantumZizo** [comment]
It seems it's actually the plasma that's the problem, not the steam
The plasma entities must be lagging it out
